content/src/content/jcr_root/apps/core/wcm/components/contentfragmentlist/v1/contentfragmentlist/clientlibs/editor/js/contentfragmentlist.js

```diff
@@ -112,29 +137,45 @@
     };
 
     /**
-     * Retrieves the html for element names and variation names and keeps the fetched values as "fetchedState" member.
+     * Creates an http request object for retrieving fragment's order by properties and returns it.
+     *
+     * @returns {Object} the resulting request object
+     */
+    ContentFragmentListController.prototype.prepareOrderByRequest = function() {
+        var data = {
```

Comment:
Seems a repeat of `ContentFragmentListController.prototype.prepareElementNamesRequest` - we could pass the URL as parameter.